AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/ManualNativeAdActivity.java

```diff
@@ -92,6 +92,12 @@ public void onNativeAdClicked(final MaxAd ad)
             {
                 logAnonymousCallback();
             }
+
+            @Override
+            public void onNativeAdExpired(final MaxAd ad)
```

Comment:
@ritamsarmah - Since our SDK does not automatically destroy ads when expired, perhaps we should include the following logic here:

- Reload new ad
- Destroy old one